app/styles/au/_au-components.scss

```diff
@@ -194,14 +200,15 @@ $au-button-height: 3.4rem;
   }
 }
 
-.au-c-button {
+.au-c-button,
```

Comment:
Is the addition of this code (and below this in the same file) perhaps a faulty merge? Because these are a practically a rollback of my changes in the previous weeks.

<img width="1624" alt="CleanShot 2023-01-19 at 20 48 10@2x" src="https://user-images.githubusercontent.com/18174827/213545149-59f873ac-5e98-4db2-b89b-55ad8d54440d.png">

Reply:
I just copied the file from `usertest` because the style was not the same as the prototype, but I could probably just change the imports on this file, leaving the rest unchanged.

Reply:
For `_au-components.scss` (and other files than styling within `prototype/*`) I think it is better to check for individual changes because `usertest` is most of the time behind the `development` branch. A sync from `development` to `usertest` is only being done from time to time due to complexity.

Reply:
Note: Don't forget to also remove this block of styling: https://github.com/kanselarij-vlaanderen/frontend-kaleidos/pull/1607/files#diff-9d68c3b32826aca3592c1e6edcadd45dda5545e67f7abc635b6a7ce97aa58528R259 as this component does not exist anymore.